src/geocoder.js

```diff
@@ -28,14 +37,18 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
```

Comment:
A maximum is needed then. Don’t implement in this PR, it’s a new feature.

When implementing a max: on error, remember that. When the last ~5 requests failed with that `OVER_QUERY_LIMIT ` error, quit with a error.